### PR TITLE
Add internationalization support and Spanish (es_ES) translation

### PR DESCRIPTION
**Pull request by jorgesumle**
The file build.py (by @onpon4) is under the GPLv3, I'm not sure how you want to
deal with that or if you @radomirbosak prefer other way to generate the `.mo`
files. That needs to be solved before merging this.

### CODE REVIEW

**Comment by radomirbosak** [comment]
Hi @jorgesumle . Thanks for submitting this PR.

Internationalisation support sounds useful to me. But I'm afraid I've never worked with `gettext` and localization - how do I test these proposed changes? Is it enough to set the `LANG=es_ES.UTF-8` environment variable?

I tried running the main entrypoint and it seems that somehow the `_` function wasn't registered. Any idea why?

```console
$ duden Loeffel
Traceback (most recent call last):
  File "/home/lukoa/.local/share/virtualenvs/duden-2j2V-L62/bin/duden", line 11, in <module>
    load_entry_point('duden', 'console_scripts', 'duden')()
  File "/home/lukoa/repos/duden/duden/main.py", line 593, in main
    args = parse_args()
  File "/home/lukoa/repos/duden/duden/main.py", line 478, in parse_args
    help=_('Display word and its article'))
NameError: name '_' is not defined
```

As for the licences - I'd prefer to keep the current license. Which project is the `build.py` script from? It seems very simple - we could have a shell script with the same functionality.

**Comment by jorgesumle** [comment]
My mistake, _ shouldn't be in the main function, but at the top I think. Later, I'll fix it.

File: duden/locale/duden.pot [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: pygettext.py 1.5\n"
+
+
+#: main.py:76
```
[the remark this answers is not in the document]
Comment (jorgesumle) [reply duden/locale/duden.pot]: You must update it only when you add new text or the original text changes: `pygettext -d duden -p locale main.py`. The line numbers are not important, as you can see, they are commented with `#`.

**Comment by jorgesumle** [comment]
> The packaging Makefile target sounds good. Did you have something like this in mind?

Sure, but I changed the build script, so that you don't have to `cd` and then `cd $OLDPWD` which is needed to be in the same directory for `make packaging`, I hope you like this improvement.

I noticed that when you generate the package with Python 2 (in my system `python` is python 2 and `python3` is Python 3), you get an encoding error because your name in setup.py has non-ascii characters (Radomír Bosák), I didn't specify the encoding with `# -*- coding: utf-8 -*-` because I assume you use Python 3 and Python 2 is about to be obsolete. If you want, I can add it.

**Comment by radomirbosak** [comment]
No need to add the encoding part.

The PR looks good :) I'll merge it.

**Comment by radomirbosak** [comment]
Merged. Thanks again for the PR :)

**Pull request merged**